.github/workflows/ci.yaml

```diff
@@ -30,17 +29,17 @@ jobs:
     strategy:
       matrix:
         # Always keep a window of "5 most recent" including (or on top with) HEAD
-        ruby: ["2.6.10", "2.7.6", "3.0.4", "3.1.2", "latest"]
-        sidekiq: ["3.3.1", "3.x", "4.x", "5.x", "6.x", "latest"]
+        ruby: ["2.7.8", "3.0.6", "3.1.4", "3.2.2", "latest"]
```

Comment:
When you are using latest version of Rails or Ruby it would be helpful to run test on schedule, monthly or biannually, just to anticipate problems before someone creates a PR.

https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#schedule

Reply:
Yeah, we can do that